src/Resources/contao/classes/Backend.php

```diff
@@ -886,9 +886,11 @@ public static function addPagesBreadcrumb($strKey='tl_page_node')
 		// Insert breadcrumb menu
 		$GLOBALS['TL_DCA']['tl_page']['list']['sorting']['breadcrumb'] .= '
 
-<ul id="tl_breadcrumb">
-  <li>' . implode(' &gt; </li><li>', $arrLinks) . '</li>
-</ul>';
+<nav aria-label="' . $GLOBALS['TL_LANG']['MSC']['breadcrumbAriaLabel'] . '" role="navigation">
```

Comment:
Why did you add the `aria-label` tag? `<nav role="navigation">` should suffice for any screen reader.